core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -917,7 +941,7 @@ public static Object percent1( ValueMetaInterface metaA, Object dataA, ValueMeta
       case ValueMetaInterface.TYPE_BIGNUMBER:
         return divideBigDecimals(
           multiplyBigDecimals( metaA.getBigNumber( dataA ), new BigDecimal( 100 ), null ), metaB
-            .getBigNumber( dataB ), null );
+            .getBigNumber( dataB ) );
```

Comment:
Need to put this back - cannot change the API

Reply:
Done